src/Billing/Billing/Az.Billing.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.0.3'
+ModuleVersion = '2.0.0'
```

Comment:
please revert back to previous version, this will be upgraded during release by Azure-PowerShell team

Reply:
Sure!